ivadomed/main.py

```diff
@@ -32,38 +32,50 @@ def get_parser():
 
     # MANDATORY ARGUMENTS
     mandatory_args = parser.add_argument_group('MANDATORY ARGUMENTS')
-    mandatory_args.add_argument("-c", "--config", required=True, type=str,
-                                help="Path to configuration file.")
-
+    
     # OPTIONAL ARGUMENTS
     optional_args = parser.add_argument_group('OPTIONAL ARGUMENTS')
+
+    command_group = optional_args.add_mutually_exclusive_group(required=False)
+    
+    command_group.add_argument("--train", dest='train', action='store_true', 
+                                help="Perform training on data.")
+    command_group.add_argument("--test", dest='test', action='store_true', 
+                                help="Perform testing on trained model.")
+    command_group.add_argument("--segment", dest='segment', action='store_true', 
+                                help="Perform segmentation on data.")
+    mandatory_args.add_argument("-c", "--config", required=True, type=str,
```

Comment:
If the config file is still part of the mandatory args, we should probably put it at the top with the other mandatory args?

Reply:
I placed them in this order to respect the convention of having the command argument before the config argument in CLI.

Reply:
i.e. `ivadomed --train -c path/to/config/config.json` vs. ivadomed -c path/to/config/config.json  --train

Reply:
Ah I see. Maybe if that's the case then, the headers "Optional Arguments" and "Mandatory Arguments" should be removed, since they don't correspond to the arguments anymore?

Reply:
Technically the mandatory and optional arguments are still placed within their respective groups, but the ordering in the code is not as straightforward. I could maybe instead remove the command group from the optional group and just make it its own group. Not sure if that would be more straightforward?

Reply:
Removing the mandatory args actually might be the simplest way around this. The only advantage of keeping of it would be in case we decide to support other mandatory args later on

Reply:
Right yah, I think the command group is okay because it's optional? It's just that because the config argument is now in the code block under the header ``# OPTIONAL ARGUMENTS``, the header for that section of code doesn't totally make sense now.

Reply:
I reorganized the parser arguments to try and make clearer

Reply:
Okay great!